src/helix-ui/elements/HXSearchAssistanceElement.js

```diff
@@ -36,14 +36,14 @@ export class HXSearchAssistanceElement extends HXElement {
         return [ 'open' ];
     }
 
-    attributeChangedCallback (attr, oldVal, newVal) {
+    $onAttributeChange (attr, oldVal, newVal) {
         let isOpen = (newVal !== null);
         if (newVal !== oldVal) {
             this.$emit(isOpen ? 'open' : 'close');
         }
-    }
+    }//$onAttributeChange()
 
-    connectedCallback () {
```

Comment:
Also, shouldn't there be an `onDisconnect` method since the `HXSearch` element gets disconnected? Or does the search assistance stay connected to the DOM for the length of the page's load time even though search does not?